### PR TITLE
review of the cloudified SF text

### PR DESCRIPTION
**Pull request by boucadair**
SF is used rather than NF for doc consistency and also to take into account rtg-dir review

### CODE REVIEW

**Review by robric — COMMENTED** [review]
I really don't like SF - it does not resonate with the  use case imho. NF is better since it has broad industry adoption.

**Comment by boucadair** [comment]
> I really don't like SF - it does not resonate with the use case imho. NF is better since it has broad industry adoption.

I suspected that but we need to be consistent though the doc and also reuse the terminology in the IETF. Thanks.

File: draft-ietf-opsawg-teas-attachment-circuit.md [hunk 1]
```diff
@@ -1250,59 +1250,54 @@ The attachment circuit in this case use a SAP identifier to refer to the physica
...
 {::include-fold ./json-examples/svc/ac-cloud-parent.json}
 ~~~~
-{: #parent-profile title="Message body for configuration of the NF ACs"}
+{: #parent-profile title="Message Body for the Configuration of The SF ACs"}
 
-Assuming a failure of compute-01, the instance nf-up-1 is redeployed to compute-07 by the NF/Cloud Orchestration. Additionally, the nf can be scaled-out thanks to the creation of an extra instance nf-up7 on compute-08. Since networking is pre-provisionned, these operations happen without any API call. In other words, this is transparent from the perspective of the configuration of the provider network.
+Assuming a failure of "compute-01", the instance "nf-up-1" can be redeployed to "compute-07" by the SF/Cloud Orchestration. Additionally, the SF can be scaled-out thanks to the creation of an extra instance "nf-up7" on "compute-08". Since connectivity is pre-provisioned, these operations happen without any API calls. In other words, this redeployment is transparent from the perspective of the configuration of the provider network.
 
 ~~~~ aasvg
 {::include-fold ./figures/ac-parent-nf-lcm.txt}
```
Comment (boucadair) [inline draft-ietf-opsawg-teas-attachment-circuit.md:1296]: What does it means "DOWN" in the figure?

Comment (robric) [reply draft-ietf-opsawg-teas-attachment-circuit.md:1296]: it is broken (failure ?)

Comment (robric) [reply draft-ietf-opsawg-teas-attachment-circuit.md:1296]: I still really  hate SF - outside classics such as Dune, Old Star Wars :-)  - 
Afaik, NF is used in IETF: https://datatracker.ietf.org/doc/html/draft-ietf-teas-5g-ns-ip-mpls

Comment (boucadair) [reply draft-ietf-opsawg-teas-attachment-circuit.md:1296]: ;-) 

I have a consistency concern here. Using SF makes my life easier. This is an appendix after all. I will see how to reconcile all views here.

File: draft-ietf-opsawg-teas-attachment-circuit.md [hunk 2]
```diff
@@ -144,7 +144,7 @@ When a customer requests a new value-added service, the service can be bound to
...
 
-This document specifies a YANG service data model ("ietf-ac-svc") for managing attachment circuits that are exposed by a network to its customers, such as an enterprise site, an SF, a hosting infrastructure, or a peer network provider. The model can be used for the provisioning of ACs prior or during advanced service provisioning (e.g., Network Slice Service {{?RFC9543}}).
+This document specifies a YANG service data model ("ietf-ac-svc") for managing attachment circuits that are exposed by a network to its customers, such as an enterprise site, an NF, a hosting infrastructure, or a peer network provider. The model can be used for the provisioning of ACs prior or during advanced service provisioning (e.g., Network Slice Service {{?RFC9543}}).
```
Comment (boucadair) [inline draft-ietf-opsawg-teas-attachment-circuit.md:147]:
```suggestion
This document specifies a YANG service data model ("ietf-ac-svc") for managing attachment circuits that are exposed by a network to its customers, such as an enterprise site, an SF, a hosting infrastructure, or a peer network provider. The model can be used for the provisioning of ACs prior or during advanced service provisioning (e.g., Network Slice Service {{?RFC9543}}).
```

File: draft-ietf-opsawg-teas-attachment-circuit.md [hunk 3]
```diff
@@ -1250,59 +1250,56 @@ The attachment circuit in this case use a SAP identifier to refer to the physica
...
 
-## Connectivity of Cloudidified Network Functions
+## Connectivity of Cloudified Network Functions
 
-This example demonstrates how the AC service model permits to manage connectivity requirements for complex Network Functions - containerized or virtualized -  that typically met in Telco Networks. This integration leverages the concept of "parent ACs" to decouple physical and logical connectivity so that several AC can shares L2 and L3 resources.This approach provides flexibility, scalability and API stability.
+This section demonstrates how the AC service model permits to manage connectivity requirements for complex Network Functions (NFs) - containerized or virtualized -  that are typically deployed in Telco networks. This integration leverages the concept of "parent AC" to decouple physical and logical connectivity so that several ACs can shares Layer 2 and Layer 3 resources. This approach provides flexibility, scalability, and API stability.
 
-The Network Function have the following characteristics:
+> NFs is used to refer to the SFs defined {{?RFC7665}}.
```
Comment (boucadair) [inline draft-ietf-opsawg-teas-attachment-circuit.md:1257]:
```suggestion
> NF is used to refer to the SFs defined {{?RFC7665}}.
```

**Pull request merged**